crates/sui-indexer/src/apis/governance_api_v2.rs

```diff
@@ -261,6 +291,19 @@ async fn exchange_rates(
     Ok(exchange_rates)
 }
 
+/// Cache a map representing the validators' APYs for this epoch
```

Comment:
Mostly for my own education -- can you explain why we need to create a singleton map here?

Reply:
My initial thinking was that this will actually keep a map with all the epochs (for historical data), but realized that this is just one epoch at a time and I guess I did not change it.

Reply:
Got it -- so potentially something that can be simplified.